### ISSUE TITLE
Tabs On Bottom not working with Firefox Portable 133

### ISSUE THREAD

**Issue by Shadowblight** [MrOtherGuy/firefox-csshacks#481]
I'm not clear if this is because it's Firefox Portable, but the Reddit threads confirming it is working perfectly fine without mention of it, implies that it likely is.

I have always used FF Portable since ages ago now. I had the v1 of this working prior to the FF 133 update. For trying to get v2 to work, I've nuked my addons/extensions. I've tried a new profile. No I don't have extra virtual desktops open currently (though that is hilariously what caused my X to close the window to forget the tabs I had open). I've nuked the userChrome.css file completely and used the pristine base of just the v2 of the CSS from here. And nothing seems to get it to work.

This is driving my absolutely batty. I will take any and all suggestions for troubleshooting at this point. Kindly help, please.

**Comment by MrOtherGuy** [comment]
I'd suggest trying some css that definitely should work without having anything else in the userChrome.css file:

```css
#TabsToolbar{ background: #f00 !important }
```

Now, if tabs toolbar doesn't become red then that probably means that Firefox is not even loading userChrome.css in the first place.

**Comment by Shadowblight** [comment]
Okay so, I figured it out finally. I apparently had an older userChrome.css in the main folder, instead of going one deeper into the chrome folder inside, and that's the one I had edited. So once I put it in the chrome folder it worked. Auditing that old one right the fuck out now so I don't hopefully confuse myself later, lol. This absolutely got my pointed in the right direction, thank you so very much.
